ReactAndroid/src/main/java/com/facebook/react/views/text/ReactFontManager.java

```diff
@@ -80,6 +83,27 @@ public static ReactFontManager getInstance() {
     String fontFamilyName,
     int style,
     Context context) {
+
+    if (!mTypeCacheLoaded) {
+      Resources resources = context.getResources();
+      int fontsId = resources.getIdentifier("fonts", "array", context.getPackageName());
```

Comment:
Unfortunately, even 1 call to getIdentifier() would cause big regression. So I don't think we can do this without any ability to disable to lookup (e.g. via provider model I described in https://github.com/facebook/react-native/pull/23865/files#r278373255

Reply:
removed all getIdentifier() calls